app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
```

Comment:
- [ ] Please There should be a form in the views/posts/new.html.erb file which accepts input for creating posts I can see here you just have two erb files for showing all the posts and for showing the post.
![image](https://user-images.githubusercontent.com/74744787/216737153-4523c429-c6b6-4d13-a225-4b1791d70824.png)
![image](https://user-images.githubusercontent.com/74744787/216737256-d154dde1-b1cf-4b48-90ba-50ac1b7f2b34.png)